keps/sig-storage/20190204-kubernetes-csi-release-process.md

```diff
@@ -7,18 +7,15 @@ participating-sigs:
   - sig-testing
   - sig-release
 reviewers:
-  - @msau42
+  - "@msau42"
 approvers:
-  - @msau42
-  - sig-testing: TBD
-  - sig-release: TBD
-editor: @pohly
+  - "@msau42"
+  - "sig-testing: TBD"
+  - "sig-release: TBD"
+editor: "@pohly"
 creation-date: 2019-02-04
 last-updated: 2019-02-04
 status: provisional
```

Comment:
is it implementable/implemented?

Reply:
I think "provisional" is more suitable. The text should be updated once more to accommodate the latest proposal for image publishing from Prow (which just came in an is still pending in Kubernetes itself: https://github.com/kubernetes/test-infra/issues/13332#issuecomment-529938165); then it can be marked "implementable".